docs/guide/helpers.md

```diff
@@ -51,6 +51,24 @@ class UserComponent < ViewComponent::Base
 end
 ```
 
+## UseHelpers setter
+
+To use the `UseHelpers` setter include `ViewComponent::UseHelpers`
+`UseHelpers` defines the helper on the component and is similar in use to using `delegate` on helpers
+
+```ruby
+class UseHelpersComponent < ViewComponent::Base
+  include ViewComponent::UseHelpers
+  use_helpers :message
```

Comment:
It's a bit unclear what this `message` helper is and where it comes from.

Reply:
@spone just edited it to use the icon example